src/OpenApi/sample/EndpointRouteBuilderExtensions.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+internal static class OpenApiEndpointRouteBuilderExtensions
+{
+    /// <summary>
+    ///  Helper method to render Swagger UI view for testing.
+    /// </summary>
+    public static IEndpointConventionBuilder MapSwaggerUi(this IEndpointRouteBuilder endpoints)
+    {
+        return endpoints.MapGet("/swagger/{documentName}", (string documentName) => Results.Content($$"""
```

Comment:
Shouldn't this be configurable? Same for the Jason document uri below

Reply:
The `MapSwaggerUi` method you see here is not a public API that we intend on exposing. It's just part of the sample app that I'm using to validate the API.

The route that is used in the `MapOpenApi` method call is configurable. You can do:

```
app.MapOpenApi();
```

or

```
app.MapOpenApi("/{documentName/openapi.json");
```